tests/bootstrap.php

```diff
@@ -19,65 +19,101 @@
 // Give access to tests_add_filter() function.
 require_once $_tests_dir . '/includes/functions.php';
 
-tests_add_filter(
-	'setup_theme',
-	function() {
-		echo esc_html( 'Installing WooCommerce...' . PHP_EOL );
-		WC_Install::install();
-		$GLOBALS['wp_roles'] = null; // WPCS: override ok.
-		wp_roles();
-	}
-);
-
 /**
- * Manually load the plugin being tested.
+ * Returns WooCommerce main directory.
+ *
+ * @return string
  */
-tests_add_filter(
-	'muplugins_loaded',
-	function() {
-		define( 'WC_TAX_ROUNDING_MODE', 'auto' );
-		define( 'WC_USE_TRANSACTIONS', false );
-
-		echo esc_html( 'Loading WooCommerce plugin' . PHP_EOL );
+function wc_dir() {
+	static $dir = '';
+	if ( $dir === '' ) {
 		if ( file_exists( WP_CONTENT_DIR . '/woocommerce/woocommerce.php' ) ) {
-			require WP_CONTENT_DIR . '/woocommerce/woocommerce.php';
+			$dir = WP_CONTENT_DIR . '/woocommerce';
 			echo "Found WooCommerce plugin in content dir." . PHP_EOL;
 		} elseif ( file_exists( dirname( dirname( __DIR__ ) ) . '/woocommerce/woocommerce.php' ) ) {
-			require dirname( dirname( __DIR__ ) ) . '/woocommerce/woocommerce.php';
+			$dir = dirname( dirname( __DIR__ ) ) . '/woocommerce';
 			echo "Found WooCommerce plugin in relative dir." . PHP_EOL;
 		} elseif ( file_exists( '/tmp/wordpress/wp-content/plugins/woocommerce/woocommerce.php' ) ) {
-			require '/tmp/wordpress/wp-content/plugins/woocommerce/woocommerce.php';
+			$dir = '/tmp/wordpress/wp-content/plugins/woocommerce';
 			echo "Found WooCommerce plugin in tmp dir." . PHP_EOL;
 		} else {
 			echo "Could not find WooCommerce plugin." . PHP_EOL;
 			exit( 1 );
 		}
+	}
+	return $dir;
+}
+
+/**
+ * Install WC Blocks
+ */
+function wc_blocks_install() {// Clean existing install first.
```

Comment:
Can you add a space here?
```suggestion
function wc_blocks_install() { // Clean existing install first.
```

Reply:
doh, this is a comment that was supposed to be deleted instead (used to be other code here that was moved elsewhere).

Reply:
fixed in f3049e0